docs/tutorials/getting_started.rst

```diff
@@ -218,469 +144,31 @@ Basic information about the file is obtained by either by ``fh.info`` or simply
     samples_per_frame = 20000
     sample_shape = (8, 1)
 
-    >>> fh
```

Comment:
Since you mention that just printing it is useful, maybe keep this?

Reply:
Sure, added it back in.